### PR TITLE
fix: do not propagate events handled by keyboard bindings

### LINKED ISSUE #3016: Undo removes multiple execution listeners at a time
### Describe the bug

When I add multiple execution listeners to a C7 BPMN diagram and press CMD+Z to undo, it removes multiple (not all) listeners and not only the last one added.

Kapture 2022-07-08 at 11 17 38

### Steps to reproduce

1. Select a service task
2. Add multiple execution listeners
3. Undo changes (e.g. via CTRL/CMD+Z)
4. See multiple listeners were removed, not only the last one added

### Expected behavior

Only the last added listener should be removed in this scenario. 

Note that it works as expected for other list groups (e.g. Inputs and Outputs).

### Environment

- OS: MacOS 11.6.5
- Camunda Modeler Version: 5.1.0-rc.1, 5.0.0
- Execution Platform: Camunda Platform 7.15
- Installed plug-ins: Cloud Connect Modeler Plugin

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by philippfromme**
Temporary fix until https://github.com/bpmn-io/bpmn-js-properties-panel/pull/739 arrives.

---

Closes #3016

### CODE REVIEW

File: client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js [hunk 1]
```diff
@@ -8,77 +8,82 @@
...
-
-function isUndo(event) {
-  return isCommandOrControl(event) && !isShift(event) && isKey([ 'z', 'Z' ], event);
-}
-
-function isRedo(event) {
-  return isCommandOrControl(event) && (isKey([ 'y', 'Y' ], event) || (isKey([ 'z', 'Z' ], event) && isShift(event)));
-}
\ No newline at end of file
+import {
```
Comment (nikku) [inline client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js:11]: What happened here? Is this now windows line endings? 

Or are the line endings fixed now? Please ship that as a separate commit. Thanks!

**Review by nikku — CHANGES_REQUESTED** [review]

File: client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js (hunk 1 above)
Comment (philippfromme) [reply client/src/app/tabs/bpmn/modeler/features/properties-panel-keyboard-bindings/PropertiesPanelKeyoardBindings.js:11]: My editor converted these old CRLF files to LF automatically.

**Review by nikku — CHANGES_REQUESTED** [review]
Tests seem to fail, please have a look.

**Review by nikku — APPROVED** [review]
:+1:

**Pull request merged**